frontend/src/store/reducers.ts

```diff
@@ -39,4 +47,12 @@ export const rootReducer = createReducer(initialState, {
     socketState: "closed",
     state: "login",
   }),
+  [showInfoMessage.type]: (state, _: PayloadAction<InfoMessage>) => ({
+    ...state,
+    // TODO this action shouldn't be handled by reducer but by some toasts component
```

Comment:
we could hold the notice in global state and just have a component that shows it